wiki/Tournaments/OFFC/cirno/en.md

```diff
@@ -220,6 +267,26 @@ Match time: 21 September 2024 ~ 29 September 2024
 | 身为pp刷子因没练图而成了传说 | **诟撡𤷙𣧏𡣀𠔑𡉄𫟺㻵𧂂𨾻𠪧** | [#7](https://osu.ppy.sh/community/matches/115662073) |
 | **Helicopter** | 内有小南梁 | [#8](https://osu.ppy.sh/community/matches/115576295) |
 
+### Finals
+
+Match time: 5 October 2024 ~ 8 October 2024
+
+| Team 1 | Team 2 | Match link |
+| --: | :-- | :-- |
+| **这是什么** | 沟槽的烤翅还在开战舰追我 | [#9](https://osu.ppy.sh/community/matches/115759584) |
+| Pingu\~Pingu\~ | **身为pp刷子因没练图而成了传说** | [#10](https://osu.ppy.sh/community/matches/115759876) |
+| **Bingo!** | 内有小南梁 | [#11](https://osu.ppy.sh/community/matches/115744612) |
+| 身为pp刷子因没练图而成了传说 | **Bingo!** | [#12](https://osu.ppy.sh/community/matches/115780911) |
+
+### Grand Finals
+
+Match time: 20 October 2024 ~ 2 November 2024
+
+| Team 1 | Team 2 | Match link |
+| --: | :-- | :-- |
+| **沟槽的烤翅还在开战舰追我** | Bingo! | [#13](https://osu.ppy.sh/community/matches/115914380) |
+| **这是什么** | 沟槽的烤翅还在开战舰追我 *(FF)* | - |
```

Comment:
Is it okay to use the abbreviation "FF" directly? Are there any other wiki articles like this?